src/example/kotlin/dev/cbyrne/example/Main.kt

```diff
@@ -43,9 +57,98 @@ suspend fun main() {
+                    if (!refreshResponse.isSuccessful) throw IOException("Unexpected code ${refreshResponse.code}")
+
+                    if (refreshResponse.body == null) throw Exception("error while getting access token")
+
+                    val oauthResponse = json.decodeFromString<OAuthResponse>(refreshResponse.body!!.string())
+                    logger.info(oauthResponse)
+
+                    accessToken = oauthResponse.accessToken
+
+                    saveOAuthResponse(oauthResponse)
+                } else {
+                    accessToken = timelessResponse.accessToken
+                }
+            } catch (exception: Exception) {
+                logger.error("Refresh token invalid")
+            }
+        }
+
+        //If token could not be fetched from saved credentials (missing authentication.json or invalid refresh token) get a new one by prompting the user
+        if (accessToken == null) {
+
+            //Request authorization from the user. See https://discord.com/developers/docs/topics/rpc#authorize
+            val authorization =
+                ipc.applicationManager.authorize(scopes = arrayOf("identify", "rpc", "rpc.voice.read"), "971413122470531142")
+            logger.info("Authorization: $authorization")
+
+            val body = MultipartBody.Builder()
+                .setType(MultipartBody.FORM)
+                .addFormDataPart("client_id", clientID)
+                .addFormDataPart("client_secret", clientSecret)
+                .addFormDataPart("grant_type", "authorization_code")
+                .addFormDataPart("code", authorization.code)
+                .addFormDataPart("redirect_uri", "http://127.0.0.1")
+                .build()
+
+            val request = Request.Builder()
+                .header("Content-Type", "application/x-www-form-urlencoded")
+                .url("https://discord.com/api/v8/oauth2/token")
+                .post(body)
+                .build()
+
+            val response = okHttpClient.newCall(request).execute()
+            if (!response.isSuccessful) throw IOException("Unexpected code $response")
+
+            if (response.body == null) throw Exception("error while getting access token")
+
+            val oauthResponse = json.decodeFromString<OAuthResponse>(response.body!!.string())
+            logger.info(oauthResponse)
+
+            accessToken = oauthResponse.accessToken
+
+            saveOAuthResponse(oauthResponse)
+        }
+
+        // Authenticate with the client and get an oauth token for the currently logged-in user
+        val oauthToken = ipc.applicationManager.authenticate(accessToken)
         logger.info("Received oauth token from Discord! Expires on: ${oauthToken.expires}")
+
+        //Prints the current voice settings
+        val voiceSettings = ipc.voiceSettingsManager.getVoiceSettings()
+        logger.info("Voice Settings: $voiceSettings")
+
+        //Mutes the user
```

Comment:
Put spaces before the content of the comment. (i.e. ``//Test``should be``// Test``)